rest-api/ext/user-status.xqy

```diff
@@ -3,6 +3,8 @@ xquery version "1.0-ml";
 module namespace user = "http://marklogic.com/rest-api/resource/user-status";
```

Comment:
user-status extension is no longer used. /user/login goes to profile extension.

Maybe we should get rid of these.. :)